plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -32,6 +32,7 @@
 import io.trino.filesystem.Location;
 import io.trino.filesystem.TrinoFileSystem;
 import io.trino.filesystem.TrinoFileSystemFactory;
+import io.trino.filesystem.TrinoOutputFile;
```

Comment:
Can you make commit message of `Fix Hive write to bucketed table when native writers disabled` `Reverts "Use Hive native writers for creating empty bucket files"` - so it is obvious this is a single commit revert

Reply:
I am following the pattern from https://github.com/trinodb/trino/pull/18612
The revert commit is clearly marked as such in the message, but not in the title.

Here, I didn't expand in the commit body on why I revert, because it's all in the title/summary line.